pkg/dashboard/functiontemplates/gitfunctiontemplatefetcher.go

```diff
@@ -35,16 +40,19 @@ type GitFunctionTemplateFetcher struct {
 	ref        string
 	repository string
 	logger     logger.Logger
+	caCert     string
 }
 
 func NewGitFunctionTemplateFetcher(parentLogger logger.Logger,
 	repository string,
-	ref string) (*GitFunctionTemplateFetcher, error) {
+	ref string,
+	caCert string) (*GitFunctionTemplateFetcher, error) {
```

Comment:
Also here, please note the comment above w.r.t renaming to `git-ca-cert-contents`